### PR TITLE
fix/TBD-7269: winutils problem

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.mapr520/plugin.xml [hunk 1]
```diff
@@ -3,6 +3,13 @@
 <plugin>
     <extension point="org.talend.core.runtime.librariesNeeded">
 
+        <!-- NAME/ID/URI of winutils MUST equals to value returned by Distribution.getWinUtilsName() -->
+        <libraryNeeded
+            context="plugin:org.talend.hadoop.distribution.mapr.6.0.0"
+            id="winutils-hadoop-2.7.0.exe"
+            name="winutils-hadoop-2.7.0.exe"
+            mvn_uri="mvn:org.talend.libraries/winutils-hadoop-2.7.0/6.0.0/exe"/>
```
Comment (StefanLMNG) [inline main/plugins/org.talend.hadoop.distribution.mapr520/plugin.xml]: The mvn uri isn't right, you should have 
mvn_uri="mvn:org.talend.libraries/winutils-hadoop/2.7.0"/>
to follow the new mvn uri policy

File: main/plugins/org.talend.hadoop.distribution.mapr500/src/main/java/org/talend/hadoop/distribution/mapr500/MapR500Distribution.java [hunk 2]
```diff
@@ -56,6 +57,10 @@
 
     private static Map<ComponentType, Set<DistributionModuleGroup>> moduleGroups;
 
+    public String getWinUtilsName() {
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.mapr500/src/main/java/org/talend/hadoop/distribution/mapr500/MapR500Distribution.java]: That means we upgraded from MapR 5.0. We had undetected issue on MapR 5.x ?

File: main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml [hunk 3]
```diff
@@ -2,6 +2,12 @@
 <?eclipse version="3.0"?>
 <plugin>
    <extension point="org.talend.core.runtime.librariesNeeded">
+        <!-- NAME/ID/URI of winutils MUST equals to value returned by Distribution.getWinUtilsName() -->
+        <libraryNeeded
+            context="plugin:org.talend.hadoop.distribution.mapr.6.0.0"
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml]: 6.0.1 ?

File: main/plugins/org.talend.hadoop.distribution.mapr500/src/main/java/org/talend/hadoop/distribution/mapr500/MapR500Distribution.java (hunk 2 above)
Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.mapr500/src/main/java/org/talend/hadoop/distribution/mapr500/MapR500Distribution.java]: It should - though I did not test that

**Pull request merged**